dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnector.java

```diff
@@ -385,18 +391,20 @@ protected static class TeradataAssessmentLogsJdbcTask extends TeradataLogsJdbcTa
                 "L.UserID",
                 "L.UserName",
                 "L.UtilityByteCount",
-                "L.UtilityInfoAvailable",
-                "L.UtilityRowCount",
-                "L.VHLogicalIO",
-                "L.VHLogicalIOKB",
-                "L.VHPhysIO",
-                "L.VHPhysIOKB",
-                "L.WarningOnly",
-                "L.WDName"
+            "L.UtilityInfoAvailable",
+            "L.UtilityRowCount",
+            "L.VHLogicalIO",
+            "L.VHLogicalIOKB",
+            "L.VHPhysIO",
+            "L.VHPhysIOKB",
+            "L.WarningOnly",
+            "L.WDName"
```

Comment:
Please, revert the formatting change as it breaks the indent compared to the rows above. If it is needed, then I would recommend moving the formatting changes to a separate PR and applying it to all the strings in the `EXPRESSIONS_FOR_ASSESSMENT` array.